### PR TITLE
AzureFileCopyV6: Fix usage of wrong path for azcopy.exe 

### PR DESCRIPTION
**Pull request by v-mohithgc**
**Task name**: AzureFileCopyV6

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** https://github.com/microsoft/azure-pipelines-tasks/issues/19705

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by manolerazvan — APPROVED** [review]

**Review by v-snalawade — APPROVED** [review]

**Comment by DevTestUser123** [comment]
Still getting error:

failed with error: 'The term 'azcopy' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.' For more info please refer to https://aka.ms/azurefilecopyreadme

**Comment by DevTestUser123** [comment]
Any update on this ?

**Comment by AArnott** [comment]
@DevTestUser123 Please don't spam folks to try to get attention on issues. I for one have nothing to do with this repo and can't help.

File: Tasks/AzureFileCopyV6/Utility.ps1 [hunk 1]
```diff
@@ -244,11 +244,12 @@ function Upload-FilesToAzureContainer
             Write-Output "##[command] & azcopy copy `"$sourcePath`" `"$containerURL`" $sanitizedArguments"
```
Comment (merlynomsft) [inline Tasks/AzureFileCopyV6/Utility.ps1:244]: Updating this line is needed as well, as this code path will impact when "Enable shell tasks arguments parameter validation" is enabled.

**Pull request merged**